docs/release/release-2.0.rst

```diff
@@ -10,6 +10,7 @@ New Features
 * Change the default configuration directory to ``~/.prestoadmin`` and the
   default log directory to ``~/.prestoadmin/log``.
 * Remove the requirement for running and installing presto-admin with sudo.
```

Comment:
didn't include the bit about to install RPM's because we use sudo for just about every command we run for Presto